client/js/src/main/scala/zio/zmx/client/frontend/views/DiagramView.scala

```diff
@@ -46,7 +49,37 @@ object DiagramView {
               s"A diagram for ${cfg.title}"
             ),
             div(
-              chart.element()
+              cls := "flex",
+              chart.element(),
+              div(
+                cls := "w-1/5 p-3 ml-2",
+                form(
+                  onSubmit.preventDefault.mapTo(
+                    Command.UpdateDiagram(cfg.copy(title = zipVar.now()))
+                  ) --> Command.observer,
+                  p(
+                    cls := "flex py-5",
+                    label(cls := "w-1/3 text-gray-50 text-xl font-bold", "Title: "),
+                    input(
+                      cls := "w-2/3 rounded-xl px-3 text-gray-600",
+                      placeholder(s"${cfg.title}"),
```

Comment:
The placeholder should contain some instructions like `Enter a diagram title` and should will be displayed only when the current `value` of the input field is the empty string. If you start changing the title and i.e. delete everything to start from scratch, with this code you would still see the previous title in the input field, just as a placeholder

Reply:
There is no need to handle the empty values. The placeholder always shows the current `cfg.title`. I can confirm that from the UI